servicecontrol/migrations/retain-data-between-instances.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Retain ServiceControl data between instances
+summary: Instructions on how to retain ServiceControl data between instances.
+reviewed: 2025-06-12
+component: ServiceControl
+related:
+    - servicecontrol/migrations/replacing-audit-instances
+    - servicecontrol/migrations/replacing-error-instances
+    - servicecontrol/backup-sc-database
+redirects:
+    - servicecontrol/data-migration
+---
+
+ServiceControl, which exists to serve the management of distributed systems, is itself a distributed system. As a result, different pieces of the system manage different subsets of ServiceControl data.
+
+> [!NOTE]
+> This process is not part of standard ServiceControl instance upgrades unless noted in the [upgrade documentation](/servicecontrol/upgrades/).
+
+Situations may arise where data retention between instances may be preferable, or even necessary instead of upgrading ServiceControl. For example:
+
+- ServiceControl instances are deployed on aging hardware and need to be moved
+- ServiceControl instances are being migrated to a different kind of deployment, such as moving from on-premises to cloud-hosted servers or from ServiceControl Management Utility installation to containers
+- ServiceControl instance upgrades are not possible
+
+This document gives an overview of strategies for retaining ServiceControl data, and links to more specific information on how to accomplish data retention for each.
+
+## Overview
+
+ServiceControl data consists of audit and error message data, managed by [audit instances](/servicecontrol/audit-instances/) and [error instances](/servicecontrol/servicecontrol-instances/) respectively.
+
+[Monitoring instances](/servicecontrol/monitoring-instances/) do not use a persistent data store, and therefore have no data to retain.
+
+## Strategies
+
+> [!NOTE]
+> It is worth assessing whether audit and error message data retention is required. For scenarios where retaining audit and error message data is not required (e.g. transient or test environment data that does not merit effort to retain), these strategies are not necessary.
+
```

Comment:
I think it helps to state when to go for Backup/Restore versus Instance replacement.

Just guessing some aspects:

- Downtime
- Lift and shift
- Version upgrades
- Very long audit retention periods
- Virtualization difference (VM to container)
- OS difference (Windows 2019 to 2022)